### PR TITLE
arkade 0.8.11 (new formula)

### PR DESCRIPTION
**Pull request by max-ae**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
lgtm

**Comment by max-ae** [comment]
Thanks for the review @chenrui333 ! Just pushed a small change to clean up the completion a bit more

File: Formula/arkade.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      -s -w
+      -X github.com/alexellis/arkade/cmd.Version=#{version}
+      -X github.com/alexellis/arkade/cmd.GitCommit=#{Utils.git_head}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    bin.install_symlink "arkade" => "ark"
+
+    (zsh_completion/"_arkade").write Utils.safe_popen_read("#{bin}/arkade", "completion", "zsh")
+    (bash_completion/"arkade").write Utils.safe_popen_read("#{bin}/arkade", "completion", "bash")
```
Comment (branchvincent) [inline Formula/arkade.rb:22]: let's also add completions for `fish`
```suggestion
    (zsh_completion/"_arkade").write Utils.safe_popen_read(bin"arkade", "completion", "zsh")
    (bash_completion/"arkade").write Utils.safe_popen_read(bin/"arkade", "completion", "bash")
    (fish_completion/"arkade.fish").write Utils.safe_popen_read(bin/"arkade", "completion", "fish")
```

Comment (max-ae) [reply Formula/arkade.rb:22]: I would have added them in the first place, but believed this command output that fish is not supported: 
```bash
$ arkade completion
Use "arkade completion SHELL" to generate SHELL completion for:
    - bash
    - zsh
```
fish works too, though 😄

**Review by branchv — APPROVED** [review]
@max-ae thank you!

**Pull request closed without merging**